### PR TITLE
Updated dependencies and base layer for openshift container

### PR DESCRIPTION
**Pull request by ryanloney**
Updated some of the dependencies used in Pipfile.lock, requirements.in and Dockerfile.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -37,7 +37,7 @@ USER 1001
 RUN /tmp/scripts/assemble
 
 # These manual pip installs will be removed before final release and added to the Piplock file
-RUN pip install openvino-dev
+RUN pip install openvino-dev tensorflow~=2.4.1 tensorflow-serving-api --use-deprecated=legacy-resolver
```
Comment (helena-intel) [inline Dockerfile:40]: Why did you move the tensorflow installs from the pipfile to here?

File: .openshift/Pipfile [hunk 2]
```diff
@@ -15,12 +15,10 @@ geffnet = "==0.9.8"
 fastseg = "*"
 onnx = "*"
 ipywidgets = "*"
-torch = "==1.5.1"
-torchvision = "==0.6.1"
+torch = "<=1.7.1+cpu,>=1.5.1+cpu"
```
Comment (helena-intel) [inline .openshift/Pipfile:18]: Should we pin PyTorch to a specific version here, since it will be pinned to a specific version in the Pipfile.lock file anyway?

**Review by helena-intel — COMMENTED** [review]
Thanks Ryan! I am happy to see Jupyter Lab 3! Did you test that everything works?

File: .openshift/Pipfile (hunk 2 above)
Comment (ryanloney) [reply .openshift/Pipfile:18]: Good idea. Updating it

File: Dockerfile (hunk 1 above)
Comment (ryanloney) [reply Dockerfile]: pipenv was not able to do the dependency resolution. once we have 2021.4 this will be easier

File: .openshift/Pipfile (hunk 2 above)
Comment (ryanloney) [reply .openshift/Pipfile:18]: I could not get this to work by changing the Pipfile and requirements.in. Let's leave it like this for now. I want to address the security issues first.

**Comment by ryanloney** [comment]
Ready to merge if you are ok with us revisiting the PyTorch part later @helena-intel? I will need to update this for 2021.4 release anyway.

**Comment by helena-intel** [comment]
> Ready to merge if you are ok with us revisiting the PyTorch part later @helena-intel? I will need to update this for 2021.4 release anyway.

Yes, the Pytorch version is not important, thanks, I'll merge this.

**Review by helena-intel — APPROVED** [review]
